src/App.jsx

```diff
@@ -26,6 +27,7 @@ function App() {
         <Route path="/book-catalog" element={<BookCatalogPage />} />
         <Route path="/book-issue" element={<BookIssuePage />} />
         <Route path="/member" element={<MemberPage />} />
+        <Route path="/ViewBookCatalog/:id" element={<ViewBookCatalog />} />
```

Comment:
Route should be like this
`<Route path="/book-catalog/:id" element={<ViewBookCatalog />} />`